sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeActionsResult.cs

```diff
@@ -11,19 +11,19 @@ namespace Azure.AI.TextAnalytics
     public class AnalyzeActionsResult
     {
         internal AnalyzeActionsResult(
-            IReadOnlyCollection<ExtractKeyPhrasesActionResult> extractKeyPhrasesActionResult,
+            IReadOnlyCollection<ExtractKeyPhrasesActionResult> extractKeyPhrasesActionResults,
             IReadOnlyCollection<RecognizeEntitiesActionResult> recognizeEntitiesActionResults,
             IReadOnlyCollection<RecognizePiiEntitiesActionResult> recognizePiiEntitiesActionResults,
-            IReadOnlyCollection<RecognizeLinkedEntitiesActionResult> recognizeLinkedEntitiesActionsResults,
-            IReadOnlyCollection<AnalyzeSentimentActionResult> analyzeSentimentActionsResults,
-            IReadOnlyCollection<ExtractSummaryActionResult> extractSummaryActionsResults)
+            IReadOnlyCollection<RecognizeLinkedEntitiesActionResult> recognizeLinkedEntitiesActionResults,
+            IReadOnlyCollection<AnalyzeSentimentActionResult> analyzeSentimentActionResults,
+            IReadOnlyCollection<ExtractSummaryActionResult> extractSummaryActionResults)
```

Comment:
Changing parameter names for consistency (we had suffixes "ActionResult", "ActionResults", and "ActionsResults"). Constructor is internal, so no public API changes.

Reply:
Thanks!!! I guess we went with so many renamings that these ones didn't get enough attention.